json/src/main/java/org/eclipse/ditto/json/JsonParseException.java

```diff
@@ -23,10 +23,10 @@
     /**
      * Error code of this exception.
      */
-    public static final String ERROR_CODE = "json.invalid";
+    public static final String ERROR_CODE = "json.format.invalid";
```

Comment:
Hm, did you break the error code on purpose?
Might be that users see this as API and assume that exact string.